### PR TITLE
Add Flask app sample that runs on Heroku

### PR DESCRIPTION
**Pull request by seratch**
```bash
heroku login
heroku create

export SLACK_BOT_TOKEN=xxx
export SLACK_SIGNING_SECRET=xxx
heroku config:set SLACK_BOT_TOKEN=$SLACK_BOT_TOKEN
heroku config:set SLACK_SIGNING_SECRET=$SLACK_SIGNING_SECRET

git init
git remote add heroku https://git.heroku.com/xxx.git
git checkout -b main
git add .
git commit -m'Initial commit for my awesome Slack app'
git push heroku main
```

### Category (place an `x` in each of the `[ ]`)

* [ ] `slack_bolt.App` and/or its core components
* [ ] `slack_bolt.async_app.AsyncApp` and/or its core components
* [ ] Adapters in `slack_bolt.adapter`

## Requirements (place an `x` in each `[ ]`)

Please read the [Contributing guidelines](https://github.com/slackapi/bolt-python/blob/main/.github/contributing.md) and [Code of Conduct](https://slackhq.github.io/code-of-conduct) before creating this issue or pull request. By submitting, you are agreeing to those rules.

* [x] I've read and understood the [Contributing Guidelines](https://github.com/slackapi/bolt-python/blob/main/.github/contributing.md) and have done my best effort to follow them.
* [x] I've read and agree to the [Code of Conduct](https://slackhq.github.io/code-of-conduct).
* [x] I've run `./scripts/run_tests.sh` after making the changes.

### CODE REVIEW

File: samples/heroku/main.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+@flask_app.route("/slack/events", methods=["POST"])
+def slack_events():
+    return handler.handle(request)
+
+# heroku login
+# heroku create
+# git remote add heroku https://git.heroku.com/xxx.git
+
+# export $SLACK_BOT_TOKEN=xxx
```
Comment (mwbrooks) [inline samples/heroku/main.py:30]:
```suggestion
# export SLACK_BOT_TOKEN=xxx
```
I think the `$` can be removed here

File: samples/heroku/main.py [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+# export SLACK_SIGNING_SECRET=xxx
+# heroku config:set SLACK_BOT_TOKEN=$SLACK_BOT_TOKEN
+# heroku config:set SLACK_SIGNING_SECRET=$SLACK_SIGNING_SECRET
+# git push heroku main
```
Comment (mwbrooks) [inline samples/heroku/main.py:34]: nit: do we want to say `main` or `master` here? Git uses `master` by default so `main` may confuse some developer. However, most Heroku users should be familiar with git and know the branch that they want to push.

**Review by mwbrooks — CHANGES_REQUESTED** [review]
I've never seen a Python app setup for Heroku - very cool @seratch!

Left some minor feedback and suggestions.

**Comment by seratch** [comment]
Congrats for your first-ever commit in this repo, Mike!

**Pull request merged**